corehq/apps/userreports/dbaccessors.py

```diff
@@ -46,6 +46,18 @@ def get_registry_report_configs_for_domain(domain):
     )
 
 
+def get_report_and_registry_report_configs_for_domain(domain):
```

Comment:
I said I'd put this in util, but I thought it made more sense to put this here instead.